delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -38,20 +39,25 @@ node {
 	def domainConfig
 	def cloudfrontEnabled
 	def flowType
+    def scm                        = env.JAZZ_SCM
 	def repo_protocol				= "http://"
 	def var_credentialsId 			= env.REPO_CREDENTIAL_ID
-	def var_bitbucket_base			= env.REPO_BASE
-	def scm_base_url 				= repo_protocol + var_bitbucket_base + "/scm/" + env.REPO_CORE + "/"
+	def var_bitbucket_base			= env.REPO_BASE	
 	def var_api_key 				= env.API_KEY
 	def var_cas_rest_repo 			= repo_protocol + var_bitbucket_base  + "/rest/api/1.0/projects/" + env.REPO_LOC + "/repos"
-	def var_bitbucket_email 		= env.REPO_EMAIL
-	def var_cas_clone_url	 		= var_bitbucket_base + "/scm/" + env.REPO_LOC + "/"
-	def configPackBBURL = "http://" + var_bitbucket_base + "/scm/" + env.REPO_CORE + "/serverless-config-pack.git"
-	def var_serviceonboarding_repo 	= repo_protocol + var_bitbucket_base + "/scm/" + env.REPO_CORE + "/service-onboarding-build-pack.git"
+	def var_bitbucket_email 		= env.REPO_EMAIL	
 	def var_bitbucket_branch_perm	= repo_protocol + var_bitbucket_base + "/rest/branch-permissions/2.0/projects/" + env.REPO_LOC + "/repos/"
 	def var_bitbucket_webhook_api	= repo_protocol + var_bitbucket_base + "/rest/webhook/1.0/projects/" + env.REPO_LOC + "/repos/"
 	def var_scm_source_hook 		= JenkinsLocationConfiguration.get().getUrl()+"/bitbucket-scmsource-hook/notify"
-	def jazzBuildModuleURL = "http://${var_bitbucket_base}/scm/${env.REPO_CORE}/jazz-build-module.git"
+	
+	//Setting SCM Repo
+	setSCMBase(scm, env.REPO_BASE)
+
+    def scm_base_url                = repo_protocol + scm_base + env.REPO_CORE + "/"
+    def var_cas_clone_url           = scm_base + env.REPO_LOC + "/"
+	def configPackBBURL             = "http://" + scm_base + env.REPO_CORE + "/serverless-config-pack.git"
+    def var_serviceonboarding_repo 	= repo_protocol + scm_base + env.REPO_CORE + "/service-onboarding-build-pack.git"
```

Comment:
variables naming consistency => serviceonboarding_repo

Reply:
Removed this as its not used in the code.